qt/scientific_interfaces/Muon/ALCDataLoadingPresenter.cpp

```diff
@@ -30,6 +30,8 @@ namespace {
 const int RUNS_WARNING_LIMIT = 200;
 // must include the "."
 const std::vector<std::string> ADDITIONAL_EXTENSIONS{".nxs", ".nxs_v2", ".bin"};
+
+bool is_decimal(const char character) { return (character == '.') ? true : false; }
```

Comment:
`return character == '.'` is sufficient.

Reply:
```suggestion
bool is_decimal(const char character) { return character == '.'; }
```